### DISCUSSION TITLE
Welcome to SafeHarbor2.0 Discussions!

### DISCUSSION THREAD

**Discussion by lex-node** [lex-node/SafeHarbor-X#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oxCraig** [comment]
Open question for others working on the proposed safe harbor:

Stepping back, are we drafting the safe harbor as a mix of maybe a Section 2(a)(1) securities-definition argument and Rule 701 and Rule 144?

The Section 2(a)(1) and Rule 701 arguments could apply pre-decentralization, then post-decentralization the Rule 144-style argument could apply.

Before decentralization, the Section 2(a)(1) argument could be that the tokens distributed to the initial development team are like partnership interests held by the partners running the business so aren’t securities. Or if the tokens are securities, a Rule 701-style argument could treat token distributions to the initial development team as equity compensation not needing registration.

Then after decentralization, the Rule 144 argument be: a cryptosystem that is or will soon be decentralized (decentralized status would be proven by meeting the proposed safe harbor’s requirements), isn’t an “issuer” so its distribution of tokens is exempt under Section 4(a)(1) as a securities transaction by “other than an issuer”.

Just asking in case analogizing to existing law/rules could help add precedential value to the proposed safe harbor, and/or help focus/speed up the drafting process.

Happy to hear others’ thoughts.

**Reply by lex-node** [reply]
thanks @oxCraig ; philosophically, your comments are very aligned with the thinking

unlike in Commissioner Peirce's proposal, there is no 'mutation doctrine' built in here and we are focused on whether certain transactions are securities transactions rather than whether certain assets (the tokens) are securities....so, as you point out, if you read through the definitions, they basically require a high degree of decentralization from the outset...

your comment does make me realize that there may be a timing or chicken/egg issue with the team allocations, since they will typically occur before the cryptosystem is live....I believe there will need to be a drafting fix for that and I will create an issue for it

**Reply by oxCraig** [reply]
Thanks @lex-node. 

Yeah, looking at the definitions it seems like there's currently the 30% cap on distributions to the dev team/insiders.

Meanwhile, elsewhere in the securities space there's the 10% shareholder control concept. So in the Web3 space, for proving absence of control / presence of decentralization for a cryptosystem, a similar type of 10% cap on holdings of insiders/dev teams/miners could be used as a bright-line rule for showing decentralization (IIRC you've done some thoughtful writing on this topic and the possible control groups). 

Maybe the % cap would be only post-decentralization (not needed pre-decentralization).
